### PR TITLE
Fixes: Improper Alignment of Boxes (Community Newsletter Box)

### LINKED ISSUE #843: Improper Alignment of Boxes
#### Description

While scrolling on the website I noticed a strange behavior in the size of the boxes (attaching the screenshot of the same)

#### Expected Behavior

The size of the boxes should be identical. (i.e Community Newsletter )

#### Screenshots

Screenshot (465)

#### Enviroment:
 - OS: Windows
 - Browser: Chrome 
 - Version: Version 105.0.5195.127 (Official Build) (64-bit)
 - Device: Desktop

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
While scrolling on the website I noticed a strange behavior in the size of the boxes (Community Newsletter).
This PR fixes #843 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by UtkarshMishra12** [comment]
I'm not able to upload the video for both before the update and after the update.
So may I upload it on the drive and paste the link here? @leecalcote @legendarykamal

File: _includes/subscribe.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% if page.title == 'Meshery | The Kubernetes and Cloud Native Manager' %}
 
 <style>
-  @media (min-width: 841px) and (max-width: 1054px) {
+  @media (min-width: 841px) and (max-width: 1210px) {
```
Comment (legendarykamal) [inline _includes/subscribe.html:4]: make it 1229px  ,a minor but still it caught my eye. 
image

**Comment by UtkarshMishra12** [comment]
Before Update - 
[Video](https://drive.google.com/file/d/1kchxaMR2mbfhkVXf-2FLc6d3qAZh2263/view?usp=sharing)

After Update - 
[Video](https://drive.google.com/file/d/1Y8hDEtKFX7-D7mE85oa9ckXH8xgGmlPs/view?usp=sharing)

**Review by legendarykamal — CHANGES_REQUESTED** [review]
good sleek @UtkarshMishra12 do this one change 👍

**Comment by UtkarshMishra12** [comment]
@legendarykamal Please check it now.

**Comment by UtkarshMishra12** [comment]
@legendarykamal  Gentle Reminder 😊

**Review by legendarykamal — APPROVED** [review]
sleek awesome good to go @UtkarshMishra12  🎸

**Comment by Krishna2323** [comment]
@UtkarshMishra12 I would suggest a change to your code.

Desc: The input box width is too small according to the width of subscribe btn (at 1229px).

Current behavior at 1229 :
Screenshot 2022-10-11 103127

Expected behavior:
Screenshot 2022-10-11 103219

**Comment by Krishna2323** [comment]
Now it looks good to me and is ready to be merged.
@legendarykamal @leecalcote, correct me if I am wrong.

File: _includes/subscribe.html [hunk 2]
```diff
@@ -1,9 +1,12 @@
 {% if page.title == 'Meshery | The Kubernetes and Cloud Native Manager' %}
 
 <style>
-  @media (min-width: 841px) and (max-width: 1054px) {
+  @media (min-width: 841px) and (max-width: 1229px) {
     .EMAIL {
-      width: 216px !important;
+      width: 350px !important;
```
Comment (legendarykamal) [inline _includes/subscribe.html:6]: cherry on the cake 🍰

**Review by legendarykamal — APPROVED** [review]

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
While you are there can you make the width of the "Project Mailing Lists" buttons consistent as well?
Also, make the width of the "subscribe" equal to the maximum width of the input field (not greater than that).

**Review by asubedy — APPROVED** [review]
LGTM 👍

**Comment by asubedy** [comment]
Could you merge the master branch to your branch it seems, and also changes from @Nikhil-Ladha, other than that it looks good. 👍

**Comment by legendarykamal** [comment]
@Nikhil-Ladha project mailing list buttons are being addressed in a separate issue and someone is working on it.

While subscribe button could be taken care of in this itself. // @UtkarshMishra12 .

**Comment by UtkarshMishra12** [comment]
@Nikhil-Ladha @legendarykamal @asubedy  Please review it now, the changes are made to the subscribe button.

**Comment by UtkarshMishra12** [comment]
Gentle reminder @Nikhil-Ladha 😊

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**